### PR TITLE
Fix: Update Kolnovel & Agitoon

### PR DESCRIPTION
**Pull request by K1ngfish3r**
Updates for Kolnovel chapter scraping and Agitoon popularNovels screen

### CODE REVIEW

File: src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js [hunk 1]
```diff
@@ -128,8 +128,9 @@ class WQMangaStreamScraper {
     let chapterText = loadedCheerio('.epcontent').html();
 
     if (sourceId === 53) {
-      let ignore = loadedCheerio('.epcontent > p').next().attr('class');
-      loadedCheerio(`p.${ignore}`).remove();
+      let ignore = loadedCheerio('article > style').text().trim().split(',');
+      ignore.push(...ignore.pop().match(/^\.\w+/));
```
Comment (rajarsheechatterjee) [inline src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js:132]:
```suggestion
      let ignore = loadedCheerio('article > style').text().trim().split(',').pop().match(/^\.\w+/);
```

Comment (K1ngfish3r) [reply src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js:132]: line 131 obtains an array of items from the 'article > style' selector.
The last element of the array contains text which are unneeded for line 133.
So I pop it, match the required pieces and push it back in

The suggested solution only obtains one item (ie: the last item), which is not enough to clean the text

**Review by rajarsheechatterjee — APPROVED** [review]

**Comment by Mingo3347** [comment]
How do I get the fixed version of kolnovel

**Comment by rajarsheechatterjee** [comment]
> How do I get the fixed version of kolnovel

You could download a dev build from [here](https://github.com/LNReader/lnreader/suites/14788417616/artifacts/841397047).

**Pull request merged**